.github/workflows/test-breaking.yaml

```
@@ -0,0 +1,17 @@
+name: Label an issue 'test-breaking' when a 'test-breaking' comment is added to a PR
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  label_issues:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    if: github.event.comment.body == 'test-breaking'
```

> This PR creates a new GH workflow. When a comment `test-breaking` is added to a PR, it sets label `test-breaking` to that PR.

If we're going to bikeshed, I'd include `robottelo` in the name (like `breaks-robottelo`) because `tests` are the ones that are in this repository itself IMHO.

I also saw in some places that a bot used `/label` as a prefix so it was clear to readers what it was supposed to do.

That would lead to
```suggestion
    if: github.event.comment.body == '/label breaks-robottelo'
```